packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -92,7 +98,13 @@ export default declare((api, options) => {
               // Otherwise, we need to memoize the context object, and change the call into a Function#call.
               // `a.?b.?()` translates roughly to `(_b = _a.b) != null && _b.call(_a)`
               const { object } = chain;
-              let context = scope.maybeGenerateMemoised(object);
+              let context = scope.maybeGenerateMemoised(
```

Comment:
Also need to sync with https://github.com/babel/babel/blob/32e7bb40279e3bc91d94cb640d24f11f632cd698/packages/babel-helper-member-expression-to-functions/src/index.js#L219